setup.cfg

```diff
@@ -79,15 +79,16 @@ test =
     docutils
     sphinx
 docs =
-    astropy>=5.0,<5.3
+    astropy
```

Comment:
Why changing the astropy version here?

Reply:
We needed these specific conditions as we were using `sphinx-astropy` but now that we do not rely on that we can use the most up to date version!

Reply:
The failing test is not associated to that? to be checked ;)

Reply:
I think the issue comes from the constraint on `numpy<2.0` as the error message is connected with multiprocessing. I have removed the constraint. Let's see.